astroid/node_classes.py

```diff
@@ -48,7 +48,6 @@
     JoinedStr,
     Keyword,
     List,
-    LookupMixIn,
     Match,
```

Comment:
We should probably mention this in the release notes.

Reply:
[b969095](https://github.com/pylint-dev/astroid/pull/2171/commits/b9690955e2006e3aef967edd5f352daae725e374)